packages/test/test-end-to-end-tests/src/test/gc/gcAttachmentBlobs.spec.ts

```diff
@@ -13,6 +13,8 @@ import { IOdspResolvedUrl } from "@fluidframework/odsp-driver-definitions";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
 import { ITestContainerConfig, ITestObjectProvider, waitForContainerConnection } from "@fluidframework/test-utils";
 import { describeNoCompat, ITestDataObject } from "@fluidframework/test-version-utils";
+// eslint-disable-next-line import/no-internal-modules
+import { BlobManager } from "@fluidframework/container-runtime/dist/blobManager";
```

Comment:
If it was a constant, it would be easy to export it but this is a static member of the class. Not sure what to to about it. @markfields do you have any ideas?

Reply:
I don't know how big of a deal it is to make this a constant, it would make it much easier for us to move out of the file.

Reply:
This is a non-blocking issue feel free to resolve if this takes up too much time

Reply:
Let's not add anything to our public export surface that we don't want to be public, just for the sake of tests. So I think either leave it as-is or just define a duplicate const in this file. No sweat duplicating it for a test because the test will fail if you get it wrong.